frontend/static/federation.json

```diff
@@ -119,7 +119,7 @@
       "i2p": ""
     },
     "testnet": {
-      "onion": "https://testraliar7xkhos2gipv2k65obykofb4jqzl5l4danfryacifi4t7qd.onion",
+      "onion": "http://testraliar7xkhos2gipv2k65obykofb4jqzl5l4danfryacifi4t7qd.onion",
```

Comment:
`https` is not valid and making the request in mobile to fail

Reply:
`https` is likely to fail in browser too, was it working in the browser? Maybe TOR browser is smart enough to replace it for `http` by default. Onion services are usually `http://` as they do not require TLS and it is, in fact, difficult to set up TLS for onion. More info: https://community.torproject.org/onion-services/advanced/https/